src/tools/ScenarioMeasurement/MemoryConsumption/IParser.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Reporting;
+using System;
+
+namespace ScenarioMeasurement
+{
+    public interface IParser
+    {
+        void EnableUserProviders(ITraceSession user);
+        void EnableKernelProvider(ITraceSession kernel);
+        IEnumerable<Counter> Parse(string mergeTraceFile, string processName, IList<int> pids, string commandLine);
+        IEnumerable<Counter> Parse(string mergeTraceDirectory, string mergeTraceFilter, string processName, IList<int> pids, string commandLine) => throw new NotImplementedException();
```

Comment:
Is this needed for something? I don't see it ever implemented/overridden. Maybe for now, it can be removed and when time comes, it will be implemented.

Reply:
This is a remnant from adding device startup parsing that appears to not have been kept, removed here.